python/valis/db/models.py

```diff
@@ -194,7 +194,8 @@ class CatalogModel(PeeweeBase):
 
 class CatalogResponse(CatalogModel, SDSSidFlatBase):
     """ Response model for source catalog and sdss_id information """
-    pass
+
+    parent_catalogs: dict[str, Any] = Field(..., description='The parent catalog associations for a given catalogid')
 
```

Comment:
I added a `field_serializer` and checked that it works. I consider explicitly adding a model with the hardcoded columns but I felt that would be annoying if we change the view because then we'd need to modify and retag Valis. But it's very simple to add a proper nested model (and the parent catalogues are not likely to change —much) so let me know if you prefer that and I'll change it.

Reply:
I think the `field_serializer` is fine.  We can add the column descriptions and metadata directly to `sdssdb`, at https://github.com/sdss/sdssdb/tree/main/schema/sdss5db/metadata, and pull them in that way.